bot.py

```diff
@@ -3,28 +3,148 @@
-ans=""
-response=""
-origQuote=""
-namelessQuote=""
-global name_alias
-name_alias = {"orange": ["nivy"], "iandur": ["lance", "lander"], "laptop monkey": ["kyo", "desktop monkey"], "emily w": ["emily", "goose"],
-"blossom": ["naaz"], "onion": ["onion", "ritvik", "aipiox"], "jc": ["jerry"], "absurdism": ["krish"], "redvilder": ["nick"],"doomgooey": ["sergey", "doom"], "jonjonnho": ["jonathan"], "みお" : ["mio"],
-"luciars": ["shiba", "lucas"], "garboguy": ["liam", "garbo" ,"mailman"], "dripbot": ["bolt"], "the-call-of-the-void": ["jessica", "obomo"], "voidlord": ["chonky","chonky birb", "StarLight "], "givemewater": ["water"], "kay911kay": ["broke&homeless", "broke", "daniel"], "salazareo": ["daniel", "salazar"], 
-"riddle": ["kana"], "theraghavsharma": ["raghav", "kyo's butler"], "winghawk": ["georges","shanker"], "lukasz345": ["lukasz"], "starlight": ["chonky","chonky birb", "voidlord"],
-"1!":["wan"], "iloveubb":["hans"], 
-"undefined_kimchi": ["kimchi", "kim"], "cluelessKimchi一IlirFan": ["kimchi", "kim"], "kimchi_bot": ["kimchi", "kim"], "Chonky Rebel BOT Kimchi": ["kimchi", "kim"], "cluelessKimchi-IlirFan": ["kimchi", "kim"]
+# The different playable games
+gameStates = {
+    'guessGame': {
+        'ans': None,
+        'origQuote': None,
+        'namelessQuote': None,
+    }
 }
 
+leaderboards = {
+    'guessGame': {
+        'userIdToPoints': {},
+        'leaderboard': []
+    }
+}
+
+
+#Display helper functions
+def DisplayHelp():
+    embed=discord.Embed(title='Guess Quote Commands!', color=0x702eb2)
+    embed.add_field(name='{0}guessquote ({0}gq)'.format(prefix), value='Starts a quote guessing game!', inline=False)
+    embed.add_field(name='{}giveup'.format(prefix), value='Ends the current game', inline=False)
+    embed.add_field(name='{}leaderboard'.format(prefix), value='displays the current leaderboard', inline=False)
+    embed.add_field(name='{}quote'.format(prefix), value='Repeats current quote', inline=True)
+    return embed
+
+def DisplayGuessGame(ans = None, origQuote = None, namelessQuote = None):
+    embed=discord.Embed(title="Guess who said this quote!", color=0x702eb2)
+    embed.add_field(name="‎", value=namelessQuote, inline=True)
+    return embed
+
+def DisplayNoGame():
+    return discord.Embed(title="Oh no!", description="There currently isn't a game running, start one with {0}guessquote or {0}gq!".format(prefix), color=0x702eb2)
+
+def DisplayGiveup(ans = None, origQuote = None, namelessQuote = None):
+    embed=discord.Embed(title="So close!", description="You weren't able to guess the quote, that sucks!", color=0x702eb2)
+    embed.add_field(name="Original Quote", value=origQuote, inline=True)
+    return embed
+
+def DisplayCorrect(ans = None, origQuote = None, namelessQuote = None):
+    embed=discord.Embed(title="Correct!", description="Spot on, you got it right!", color=0x702eb2)
+    embed.add_field(name="Original Quote", value=origQuote, inline=True)
+    return embed
+
+def DisplayIncorrect():
```

Comment:
to avoid clutter, maybe the bot could instead react to the message with an X emoji?

Reply:
Would we follow the same convention for the winner as well?